website/docs/r/compute_router.html.markdown

```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router
 
-Manages a Cloud Router resource. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router resource. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/routers).
```

Comment:
I get a 404 here - did you mean https://cloud.google.com/compute/docs/reference/latest/routers?